hadoop-ozone/dist/src/main/compose/ozone/test-ec.sh

```diff
@@ -37,6 +37,7 @@ execute_robot_test scm -v PREFIX:${prefix} -N read-4-datanodes ec/read.robot
 docker-compose up -d --no-recreate --scale datanode=3
 execute_robot_test scm -v PREFIX:${prefix} -N read-3-datanodes ec/read.robot
 docker-compose up -d --no-recreate --scale datanode=5
+execute_robot_test scm -v container:1 -v count:5 -N EC-recovery replication/wait.robot
```

Comment:
If the cluster is originally 5 nodes, then it is scaled to 4, then 3 to test the reads for online recovery. Then we scale it back to 5 - will that bring back existing nodes that already had replicas? If so, this will not really test reconstruction as it will be just the DNs reporting in with their existing replicas?

Reply:
Ah ok. I have to admit I have generally started a cluster with --scale datanodes=x, and then used stop and start. So I have learned something here. OK - makes sense then that the reconstruction will occur with scale, while it would not with stop.